roles/openshift_metrics/tasks/install_heapster.yaml

```diff
@@ -41,6 +41,8 @@
       - {port: 80, targetPort: http-endpoint}
     selector:
       name: "{{obj_name}}"
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: heapster-certs
```

Comment:
Have you tested this?  I wonder if you need to quote the key in order for Ansible to properly interpret the intention